documentation/source/LRScheduling.md

```diff
@@ -202,36 +205,89 @@ Note that internally, Trainer unpacks [training_params to the scheduler callback
 ### Using PyTorchs Native LR Schedulers (torch.optim.lr_scheduler)
 
 PyTorch offers a [wide variety of learning rate schedulers](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate).
-They can all be easily used by wrapping them up with `LRSchedulerCallback` and passing them as phase_callbacks.
+They can all be easily used by passing a Mapping through the lr_mode parameter, following aa simple API.
+From `Trainer.train(...)` docs:
+
+    When Mapping, refers to a torch.optim.lr_scheduler._LRScheduler, following the below API:
+    
+        lr_mode = {LR_SCHEDULER_CLASS_NAME: {**LR_SCHEDULER_KWARGS, "phase": XXX, "metric_name": XXX)
+    
+        Where "phase" (of Phase type) controls when to call torch.optim.lr_scheduler._LRScheduler.step().
```

Comment:
From this line it is not immediately clear what are available options are. In fact, Phase enum contains too many options. 
Usually there are only two options: batch step (E.g updating LR after each gradient update step) and epoch step (Updating LR after the epoch). I can easily see one can get lost with all values Phase enum offers here.

Reply:
So maybe would be helpful to add tips:
- I want to update LR on each batch:    Use `phase: Phase.TRAIN_BATCH_END`
- I want to update LR after each epoch: Use `phase: Phase.TRAIN_EPOCH_END`

Reply:
added your suggestion to docs in 2e9df8ac80feacac1b551dfb3c2961dcc68bd7c4